frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PTSelectAssessmentYear.js

```
@@ -0,0 +1,129 @@
+
+    // Sync local state with incoming value prop changes
+    useEffect(() => {
+        if (value !== selectedAssessmentYear) {
+            setSelectedAssessmentYear(value);
+        }
+    }, [value]);
+
+    // Sync form value with parent form data and validation for employee userType
+    useEffect(() => {
+        if (userType === "employee") {
+            let keys = Object.keys(formValue);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'keys' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)